coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
@Uran198 I agree. So the ideal solution is to match the contents of `coalahtml/_coalahtml/` and the `<target directory>`. IIUC I should better match the directory contents - filenames, file contents etc. I believe http://stackoverflow.com/a/24860799/3673031 is what I need. Thoughts? Is there any better approach?